python/cudf_polars/tests/expressions/test_agg.py

```diff
@@ -7,15 +7,20 @@
 import polars as pl
 
 from cudf_polars.dsl import expr
-from cudf_polars.testing.asserts import assert_gpu_result_equal
+from cudf_polars.testing.asserts import (
+    assert_gpu_result_equal,
+    assert_ir_translation_raises,
+)
 
 
-@pytest.fixture(params=sorted(expr.Agg._SUPPORTED))
+@pytest.fixture(
+    params=sorted(expr.Agg._SUPPORTED | expr.UnaryFunction._supported_cum_aggs)
```

Comment:
I wonder if we should prefer to spell out what's being tested here rather than relying on the attributes of the classes themselves which feels a bit circular. I worry that an entry can be deleted from the mapping breaking a feature but without resulting in a test failure.

Reply:
Good point, updated.